test/onnx/expect/TestOperators.test_slice.expect

```diff
@@ -0,0 +1,81 @@
+ir_version: 3
+producer_name: "pytorch"
+producer_version: "0.4"
+graph {
+  node {
+    input: "0"
+    output: "1"
+    op_type: "Slice"
+    attribute {
+      name: "axes"
+      ints: 0
+      type: INTS
+    }
+    attribute {
+      name: "ends"
+      ints: -1
```

Comment:
-1 is not specified in onnx, but it's a hack for the caffe2 (supported by caffe2)

Reply:
After a thought, I think it's better to keep using 9223372036854775807, and let's turn it into -1 in caffe2's onnx backend. We already have some end to end test cases. So this should be covered.